app/models/concerns/urgency.rb

```diff
@@ -20,8 +20,8 @@ def confirm_still_urgent
   end
 
   class_methods do
-    def urgent_patients(lines = LINES)
-      Patient.in(line: lines).where(urgent_flag: true)
+    def urgent_patients(line)
+      Patient.where(line: line).where(urgent_flag: true)
```

Comment:
TRIVIAL: Is it possible to combine the .where's into one? Not a big deal but figured I'd see.

Reply:
def!

Reply:
I think that'd be better, especially if it's doing something like && or ||, that way it's easier to read. Looking at it right now I wouldn't be able to tell that right off the bat.